src/style.css

```diff
@@ -0,0 +1,86 @@
+* {
+  margin: 0;
+}
+
+body {
+  font-family: Arial, sans-serif;
+}
+
+.wrapper {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.wrapper__block {
+  min-height: 300px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 100px;
+  color: #fff;
+  flex-basis: 33.33%;
+  text-align: center;
+}
+
+.wrapper__block:nth-child(1) {
+  background-color: rgb(100%, 0%, 0%);
+}
+
+.wrapper__block:nth-child(2) {
+  background-color: rgb(80%, 0%, 0%);
+}
+
+.wrapper__block:nth-child(3) {
+  background-color: rgb(60%, 0%, 0%);
+}
+
+.wrapper__block:nth-child(4) {
+  background-color: rgb(40%, 0%, 0%);
+  order: 6;
+}
+
+.wrapper__block:nth-child(5) {
+  background-color: rgb(20%, 0%, 0%);
+  order: 5;
+}
+
+.wrapper__block:nth-child(6) {
+  background-color: rgb(0%, 0%, 0%);
+  order: 4;
```

Comment:
you can star from 1

Reply:
dont understand

Reply:
You don't have 1, 2, 3 orders. It's not a mistake, but try not to use too big values )